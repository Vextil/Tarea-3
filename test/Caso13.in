# stress Pila
# Prueba de orden de tiempo
TiempoPila
# Prueba de uso de memoria
MemoriaPila
Fin
