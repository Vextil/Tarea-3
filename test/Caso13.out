1># stress Pila
2># Prueba de orden de tiempo
3>Fin Apilar. 
4># Prueba de uso de memoria
5>Fin Destruir. Fin Desapilar. 
6>Fin
